### ISSUE TITLE
Doesn't work with mariadb

### ISSUE THREAD

**Issue by sebasptsch** [ghoshnirmalya/nextjs-authentication-using-strapi-and-next-auth#2]
I've followed the guide, copy-pasting as I go with the only difference being a different database software, mariadb. 
I get the error: `ER_NO_SUCH_TABLE: Table 'strapi.accounts' doesn't exist`

**Comment by sebasptsch** [comment]
Turns out synchronize wasn't on.

**Comment by ghoshnirmalya** [comment]
@sebasptsch Can you also write a little bit about what you did? It might help someone else who is also facing the same issue.

**Comment by sebasptsch** [comment]
Initially when I setup the authentication from the blog post on the Strapi Blog I copied how the postgresql server url was set out but I wasn't sure whether sync was database dependant so I removed it. Because of this nextauth was unable to create a new table in the Strapi database and therefore errored every time to use it. To fix it I added sync back after seeing it in the next-auth documentation where it describes it's purpose as creating new tables to enter the auth data within. This allowed the next-auth to create the table and therefore resolve the error I was having.
